### DISCUSSION TITLE
Applifcation rejected with all information requited and a dated transcript

### DISCUSSION THREAD

**Discussion by Scott-STEM-OU** [community/community#141369] [GitHub Education]
### Hi everyone,

Hi there I have recently applied to join github education. After several attempts and following the guidelines as set out in your emails I am still getting a rejection letter please help. I have attached I transcript of my university for your reference.
[academic-summary.pdf](https://github.com/user-attachments/files/17357049/academic-summary.pdf)

**Comment by t1noo7** [comment]
Hi @Scott-STEM-OU ,

By having followed the required steps of the student development pack procedure and being denied, I suggest you the best and fastest way to handle it at the moment is to contact for support. By creating a ticket [here](https://support.github.com/contact/education), choose a proper response from the drop-down list as you desired, e.g: `Application not approved (student)`, tell them everything you did, everything you have the correct with, and do so send them your request.
